packages/react-components/react-card/library/src/components/CardFooter/CardFooter.tsx

```diff
@@ -4,6 +4,7 @@ import { renderCardFooter_unstable } from './renderCardFooter';
 import { useCardFooterStyles_unstable } from './useCardFooterStyles.styles';
 import type { CardFooterProps } from './CardFooter.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { useCustomStyleHook_unstable } from '../../../../../react-shared-contexts/library/src/index';
```

Comment:
```suggestion
import { useCustomStyleHook_unstable } from '@fluentui/react-shared-contexts';
```

I think this is the issue